src/test/scala/edu/ie3/simona/agent/grid/PowerFlowSupportSpec.scala

```diff
@@ -153,21 +219,47 @@ class PowerFlowSupportSpec
           )
         )(ZonedDateTime.now())
 
-        // lines that are adjacent to the open switch should have no load
-        val expectedNoLoadLines =
+        // left/top side segments (lines that are adjacent to the open switch) should have no load
+        val loadLinesLeft =
           Iterable(line18To1, line0To17, line0To15, line16To3).map(_.uuid).toSet
         pfResult.lineResults
-          .filter(lineRes =>
-            expectedNoLoadLines.contains(lineRes.getInputModel)
-          )
-          .forall(_.getiAMag().isLessThan(1e-3.asAmpere)) shouldBe true
+          .filter(lineRes => loadLinesLeft.contains(lineRes.getInputModel))
+          .foreach { lineRes =>
+            lineRes.getiAMag() should equalWithTolerance(
+              0.0001d.asAmpere,
+              currentTolerance
+            )
+            lineRes.getiBMag() should equalWithTolerance(
+              0.0001d.asAmpere,
+              currentTolerance
```

Comment:
Very minor but why don't we compare against 0 with the respective tolerance since we actually expect no load?

Reply:
@t-ober because the resulting currents are not within the tolerance range around 0